### PR TITLE
telegram desktop: update to 2.4.6

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

telegram desktop: update to 2.4.6

- Update tg_owt

Package(s) Affected
-------------------

telegram-desktop 2.4.6

Security Update?
----------------

No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by eatradish** [comment]
In AMD64, looks good to me:

image

**Comment by MingcongBai** [comment]
Looks good to me on POWER.

图片

**Review by MingcongBai — APPROVED** [review]

File: extra-web/telegram-desktop/spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
-VER=2.4.4
+VER=2.4.6
 SRCS="tbl::https://github.com/telegramdesktop/tdesktop/releases/download/v$VER/tdesktop-$VER-full.tar.gz \
-      git::rename=tg_owt;commit=c73a4718cbff7048373a63db32068482e5fd11ef::https://github.com/desktop-app/tg_owt"
```
Comment (MingcongBai) [inline extra-web/telegram-desktop/spec:3]: How is this commit determined and tracked? Couldn't this be addressed with Git submodule?

**Comment by MingcongBai** [comment]
@MingcongBai @Icenowy Could you please help testing on AArch64?

File: extra-web/telegram-desktop/spec [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
-CHKSUMS="sha256::88799708daedaf5c132752259945b4e1b5dcbb62385661e2cca7b8e5d58c9026 SKIP"
+      git::rename=tg_owt;commit=e8fcae73947445db3d418fb7c20b964b59e14706::https://github.com/desktop-app/tg_owt"
+CHKSUMS="sha256::2c6f97ca079c3f9aad700f133d8bcd45ae4cfa32423e91d02c79a163664c13a4 SKIP"
 SUBDIR=tdesktop-$VER-full
+
+# The update of tg_owt follows the latest commit of the upstream master
```
Comment (MingcongBai) [inline extra-web/telegram-desktop/spec]: Move this directly after `VER=`, above the `SRCS=` values, and please reword it as follows:

"Update tg_owt to the latest Git snapshot when updating Telegram Desktop."

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**